test/unit/VariableSupplyToken.js

```diff
@@ -3,12 +3,12 @@ import chai, {expect, assert} from "chai"
 import {solidity} from "ethereum-waffle"
 chai.use(solidity)
 
-describe("VariableSupplyToken", () => {
+describe("LivepeerToken", () => {
```

Comment:
Is this change needed?

Reply:
`VariableSupplyToken` was changed to abstract in the previous PR so need to use `LivepeerToken` now

Reply:
Ah I see. Can we change the name of this test file to `LivepeerToken.js` to reflect that the contract under test is the LivepeerToken contract?